### PR TITLE
LPS-187626 Remove no needed CSS

### PR DESCRIPTION
**Pull request by sandrodw3**
Hi guys! I'm sending this as a fix of [LPS-187626](https://liferay.atlassian.net/browse/LPS-187626).

## Context
Currently, when using `clay:tabs` taglib, the correct usage is adding a `clay:tabs-panel` inside it for each tab content.

However, using it that way causes a double tabs panel, as the `clay:tabs` taglib is rendering a `ClayTabs.Panel` for each child, and then we are also rendering the same markup inside the `clay:tabs-panel` taglib.

## Solution
What I did is just removing the tabs panel markup from `clay:tabs-panel`, and let `clay:tabs` manage it, as this is needed for the `active` state to work properly. After doing this, I thought if we actually need the `clay:tabs-panel` taglib, but I think it's a good idea to leave it so the use of the taglib is more clear.

### Note
Only https://github.com/liferay-frontend/liferay-portal/pull/3915/commits/e1aff87eb1b37cff30a739f41edc24f192c501cc needs to be reviewed, as https://github.com/liferay-frontend/liferay-portal/pull/3915/commits/66cde1b3381096135f64503ca2b9cfdbfbb8da13 only adapts some CSS and all that code belongs to us in the Echo team.

Please let me know your thoughts, thanks!

### CODE REVIEW

**Comment by dsanz** [comment]
After the call with @sandrodw3: It's true that tag emits markup that ends up being redundant once react clay component kicks in. Nevertheless, that markup is there to provide a tabs experience while the react component is being loaded and executed. If we remove it, all tab panels will be visible for some time.

Alternative is to work on `Tabs` component to manage the DOM manipulation in a way that prevents the duplicate markup to happen when clay react component kicks in

**Comment by sandrodw3** [comment]
@dsanz I have updated the PR with the approach we discussed this morning 😄 I'm now just appending the children of the tab panel element directly instead of the panel itself. This way the server side rendering works properly. Let me know your thoughts!

File: modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/Tabs.js [hunk 1]
```diff
@@ -6,12 +6,14 @@
...
 
-const Panel = ({children}) => {
+const Panel = ({children: tabPanel}) => {
 	const ref = useRef();
 
 	useEffect(() => {
-		ref.current.appendChild(children);
-	}, [children]);
+		for (const child of tabPanel.children) {
+			ref.current.appendChild(child);
```
Comment (dsanz) [inline modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/Tabs.js:14]: We discovered during testing that this is appending one of every 2 nodes. Reason is that `appendChild()` removes the node from the source nodelist (`tabPanel.children`) when appending into `ref.current` so the `tabPanel.children` iterator is being mutated while loop traverses it.

Comment (sandrodw3) [reply modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/Tabs.js]: Hi Dani! I've updated the PR with the approach we discussed yesterday, appending the first child, so it doesn't affect the loop. Let me know what you think 😄

**Comment by dsanz** [comment]
Closed in favor of https://github.com/liferay-frontend/liferay-portal/pull/3922

**Pull request closed without merging**